scripts/analysis/analyze_original_vocabulary.py

```
@@ -0,0 +1,319 @@
+import json
+import sys
+from collections import Counter, defaultdict
+from typing import Dict, List, Any, Optional
+
+
+def load_captions(filepath: str) -> List[Dict[str, Any]]:
+    """Load the captions JSON file."""
+    try:
+        with open(filepath, "r", encoding="utf-8") as f:
+            return json.load(f)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)